content/src/service/validations/Validations.ts

```diff
@@ -35,6 +38,7 @@ export class Validations {
         }. You can upload up to ${entity.pointers.length * maxSizeInBytes} bytes but you tried to upload ${totalSize}.`
       ]
     }
+    return this.WEARABLE_SIZE(args)
```

Comment:
mm, we are using return undefined as when no error is present, changing this will make it incompatible with existing validations

Reply:
`WEARABLE_SIZE` also returns `undefined` as when no error is present, I guess there's no problem with that 🤔

Reply:
I think that this method should return undefined too